ignite/contrib/handlers/tensorboard_logger.py

```diff
@@ -234,6 +231,28 @@ def global_step_transform(*args, **kwargs):
                 global_step_transform=global_step_transform
```

Comment:
The rendering for this part also seems incorrect. The line 206 (Another example ..) is being pushed before the code block and the code blocks are getting combined into one.

Reply:
Yes, docs rendering becomes incorrect in this PR: 
- master: https://pytorch.org/ignite/master/generated/ignite.contrib.handlers.tensorboard_logger.html#ignite.contrib.handlers.tensorboard_logger.OutputHandler
- PR: https://deploy-preview-2137--pytorch-ignite-preview.netlify.app/generated/ignite.contrib.handlers.tensorboard_logger.html#ignite.contrib.handlers.tensorboard_logger.OutputHandler

You can what happens locally following this guidelines: https://github.com/pytorch/ignite/blob/master/CONTRIBUTING.md#writing-documentation